src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -201,18 +161,18 @@ public bool TryGetTelemetryId(out Guid telemetryId)
                     var workspace = document.Project.Solution.Workspace;
                     var supportsFeatureService = workspace.Services.GetRequiredService<ITextBufferSupportsFeatureService>();
 
-                    var selection = TryGetCodeRefactoringSelection(range);
+                    var selection = TryGetCodeRefactoringSelection(state, range);
 
                     Func<string, IDisposable?> addOperationScope =
                         description => operationContext?.AddScope(allowCancellation: true, string.Format(EditorFeaturesResources.Gathering_Suggestions_0, description));
 
                     // We convert the code fixes and refactorings to UnifiedSuggestedActionSets instead of
                     // SuggestedActionSets so that we can share logic between local Roslyn and LSP.
                     var fixes = GetCodeFixes(
-                        supportsFeatureService, requestedActionCategories, workspace,
+                        state, supportsFeatureService, requestedActionCategories, workspace,
```

Comment:
i was about to ask why we pass state *and* workspace.  and now i see taht we have _workspace and workspace, and now i'm sad :(